apps/database-abstractor/src/main/resources/struts.xml

```diff
@@ -442,6 +442,17 @@
             </result>
         </action>
 
+        <action name="api/updateRepoLastRun" class="com.akto.action.CodeAnalysisAction" method="updateRepoLastRun">
+            <interceptor-ref name="json"/>
+            <interceptor-ref name="defaultStack" />
+            <result name="SUCCESS" type="json"/>
+            <result name="ERROR" type="json">
```

Comment:
add rbac

Reply:
this is db-abstractor, 
we don't put rbac here